ui/src/alerting/constants/index.ts

```diff
@@ -103,6 +103,17 @@ export const NEW_TAG_RULE_DRAFT: TagRuleDraft = {
   },
 }
 
+export const NEW_ENDPOINT_DRAFT: NotificationEndpoint = {
```

Comment:
For rules I ended up making `const newRuleDraft = (orgID: string) => ({orgID, ...})`, which was necessary to save the rule successfully. Seems like something similar will be needed here, but we can do it later.